File: RecipeViewerAndroidApp/RecipeViewer/app/src/main/java/com/plt3ch/recipeviewer/Models/UserFeedback.java
package com.plt3ch.recipeviewer.Models;

import java.io.Serializable;

/**
 * Created by Plamen on 2/13/2017.
 */

public class UserFeedback implements Serializable {

    private String UserId;
    private int RecipeId;
    private String Comment;
    private int Rating;

    public String getComment() {
        return Comment;
    }

    public void setComment(String comment) {
        Comment = comment;
    }

    public int getRating() {
        return Rating;
    }

    public void setRating(int rating) {
        Rating = rating;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public int getRecipeId() {
        return RecipeId;
    }

    public void setRecipeId(int recipeId) {
        RecipeId = recipeId;
    }
}

File: RecipeViewerWebService/RecipeViewer/RecipeViewer.Models/UserFeedback.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RecipeViewer.Models
{
    public class UserFeedback
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public int RecipeId { get; set; }

        public virtual Recipe Recipe { get; set; }

        public string UserId { get; set; }

        public virtual User User { get; set; }

        public int Rating { get; set; }

        public string Comment { get; set; }
    }
}
